arex-web-core/src/main/java/com/arextest/web/core/repository/mongo/util/OldDataCleaner.java

```diff
@@ -0,0 +1,71 @@
+package com.arextest.web.core.repository.mongo.util;
+
+import com.arextest.web.model.dao.mongodb.ConfigComparisonIgnoreCategoryCollection;
+import com.arextest.web.model.dao.mongodb.entity.CategoryDetailDao;
+import org.apache.commons.collections4.CollectionUtils;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.query.Criteria;
+import org.springframework.data.mongodb.core.query.Query;
+import org.springframework.stereotype.Repository;
+
+import javax.annotation.PostConstruct;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+
+/**
+ * @author wildeslam.
+ * @create 2024/1/26 11:00
+ */
+@Repository
+@Lazy(false)
+public class OldDataCleaner {
+    private final MongoTemplate mongoTemplate;
+
+    public OldDataCleaner(MongoTemplate mongoTemplate) {
+        this.mongoTemplate = mongoTemplate;
+    }
+
+    @PostConstruct
+    public void main() {
+        CompletableFuture.runAsync(() -> cleanOldData(mongoTemplate));
+    }
+
+    public void cleanOldData(MongoTemplate mongoTemplate) {
+        cleanConfigComparisonIgnoreCategoryCollection(mongoTemplate);
+    }
+
+    private void cleanConfigComparisonIgnoreCategoryCollection(MongoTemplate mongoTemplate) {
```

Comment:
Whether to consider the overall refresh logic to be executed in the database instead of in the machine memory

Reply:
‘eval’ operation is no longer supported.